### PR TITLE
Prac 9 code review request

### PR DESCRIPTION
**Pull request by chenhaoxiang-jcu**
Please review.  @ron991

### CODE REVIEW

File: prac_09/unreliable_car.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""Car class"""
+
+from random import uniform
```
Comment (ron991) [inline prac_09/unreliable_car.py:3]: Great code , randint is alternative answer

**Review by ron991 — COMMENTED** [review]
Great code, Although it was not required I am curious to know how long it took you to complete each program. In previous pracs it was asked to show the date and time  to complete.

File: prac_09/silver_service_taxi_test.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
```
[the remark this answers is not in the document]
Comment (chenhaoxiang-jcu) [reply prac_09/silver_service_taxi_test.py:2]: Great advice!

File: prac_09/taxi_simulator.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+from prac_09.taxi import Taxi
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+MENU = 'q)uit, c)hoose taxi, d)rive'
```
[the remark this answers is not in the document]
Comment (chenhaoxiang-jcu) [reply prac_09/taxi_simulator.py:4]: Yeah, I've thought this but need to match the sample output ;)

File: prac_09/silver_service_taxi_test.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (lindsaymarkward) [reply prac_09/silver_service_taxi_test.py:2]: I don't see a reason to add a main/test function when your whole file has no other function.

[the remark this answers is not in the document]
Comment (chenhaoxiang-jcu) [reply prac_09/silver_service_taxi_test.py:2]: Good morning, sir. What a honer to receive your comment, I'll keep that in my mind!😄

**Pull request merged**